pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>io.camunda.connector</groupId>
     <artifactId>connector-parent</artifactId>
-    <version>0.6.0-SNAPSHOT</version>
+    <version>0.6.0-alpha3</version>
```

Comment:
Should it be still `SNAPSHOT` in PR?

Reply:
It needs to use the latest release with the[ relevant commit](https://github.com/camunda/connector-sdk/commit/6a6b8dffb149347ad3990e0da3a6d3aec9a69c6d) to handle secrets in inherited classes.